### ISSUE TITLE
Hover-over doesn't work when there's an overlay "in the way"

### ISSUE THREAD

**Issue by dtreich-itdp** [ITDP/atlas#31]
Hovering over the different features (analysis areas) to get a tooltip with the indicator reading is a very helpful way to get a lot of data quickly. Unfortunately, the overlays seem to "block" the hover-over tooltips. The tooltip only appears when your mouse is directly over a part of the analysis area that is not within a "people near X" polygon.

**Comment by kauebraga** [comment]
Yeah, this seems to be a limitation/bug in the flatgeobuff rendering. The options are there, but they don't seem to work. I opened an [issue](https://github.com/r-spatial/leafem/issues/71) in the responsible package to see if there is anything quick we can do.
